frontend/src/main/components/Nav/AppNavbar.js

```diff
@@ -121,18 +120,6 @@ export default function AppNavbar({ currentUser, systemInfo, doLogout, currentUr
                 )
               }
             </Nav>
-              {
```

Comment:
I think this change is ok, since this is the duplicate copy of the articles menu in the navbar, right?

Reply:
Yes